### PR TITLE
#trivial MOBILE-1051 Reduce debounce timeout for autosave and evaluate context

### PR DESCRIPTION
**Pull request by phcp**
@victorg1991 @dgarciasarai

### CODE REVIEW

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/DDMFormView.kt [hunk 1]
```diff
@@ -255,7 +255,7 @@ class DDMFormView @JvmOverloads constructor(
 	override fun subscribeToValueChanged(observable: Observable<Field<*>>) {
 		subscription = observable.doOnNext {
 			presenter.fieldModelsChanged(it)
-		}.debounce(2, TimeUnit.SECONDS)
+		}.debounce(500, TimeUnit.MILLISECONDS)
```
Comment (victorg1991) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/DDMFormView.kt]: 500ms isn't too little time?

Comment (dgarciasarai) [reply android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/DDMFormView.kt]: For me it's not too little time. If in the demo worked well, we can keep it like that

Comment (victorlaerte) [reply android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/DDMFormView.kt]: We can make this parameterizable, so the user can choose the best time for him. I'll create a ticket for this.

Comment (dgarciasarai) [reply android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/DDMFormView.kt]: I like that 👍  Thanks!

Comment (victorg1991) [reply android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/DDMFormView.kt]: That solution is the most flexible and convenient, yeah. So, just merge this and add that after

**Pull request merged**